microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md

```diff
@@ -64,19 +64,19 @@ To edit (or define) ATP policies, you must be assigned an appropriate role. The
  
-- If you already have a list of URLs in your "do not rewrite" list, make sure to review that list and add wildcards as appropriate. For example, if your existing list has an entry like `http://contoso.com/a` and you want to include subpaths like `http://contoso.com/a/b` in your policy, add a wildcard to your entry so it looks like `http://contoso.com/a*`.
+- If you already have a list of URLs in your "do not rewrite" list, make sure to review that list and add wildcards as appropriate. For example, if your existing list has an entry like `http://contoso.com/a` and you want to include subpaths like `http://contoso.com/a/b` in your policy, add a wildcard to your entry so it looks like `http://contoso.com/a/*`.
     
 - Do not include a forward slash (/) in the URLs that you specify in your "do not rewrite" list. For example, rather than enter `contoso.com/` in your "do not rewrite" list, enter `contoso.com`.
 
-- When you specify a "do not rewrite" list for an ATP Safe Links policy, you can include up to three wildcard asterisks (\*). Wildcards (\*) are used to explicitly include prefixes or subdomains, like `http://` or `https://`. An entry, such as `contoso.com` is not the same as `*contoso.com*` for your "do not rewrite" list. You must have `*contoso.com*` if you want to allow people to visit a domain and its subdomains and paths.
+- When you specify a "do not rewrite" list for an ATP Safe Links policy, you can include up to three wildcard asterisks (\*). Wildcards (\*) are used to explicitly include prefixes or subdomains, like `http://` or `https://`. An entry, such as `contoso.com` is not the same as `*.contoso.com/*` for your "do not rewrite" list. You must have `*.contoso.com/*` if you want to allow people to visit a domain and its subdomains and paths.
     
 The following table lists examples of what you can enter and what effect those entries have.
     
 |**Example Entry**|**What It Does**|
 |:-----|:-----|
 |`contoso.com`|Allows recipients to visit a site like `http://contoso.com` but not subdomains or paths.|
-|`*contoso.com*`  <br/> |Allows recipients to visit a domain, subdomains, and paths, such as `http://www.contoso.com`, `https://www.contoso.com`, `https://maps.contoso.com`, or `http://www.contoso.com/a`  <br/> |
+|`*.contoso.com/*`  <br/> |Allows recipients to visit a domain, subdomains, and paths, such as `http://www.contoso.com`, `https://www.contoso.com`, `https://maps.contoso.com`, or `http://www.contoso.com/a`  <br/> |
 |`http://contoso.com/a`  <br/> |Allows specific recipients to visit a site like `http://contoso.com/a`, but not subpaths like `http://contoso.com/a/b`  <br/> |
-|`http://contoso.com/a*`  <br/> |Allows specific recipients to visit a site like `http://contoso.com/a` and subpaths like `http://contoso.com/a/b`  <br/> |
+|`http://contoso.com/a/*`  <br/> |Allows specific recipients to visit a site like `http://contoso.com/a` and subpaths like `http://contoso.com/a/b`  <br/> |
```

Comment:
I'm not sure this example is right either - surely the /* at the end means you can't go to contoso.com/a?